MVP/defaults-build.yaml

```diff
@@ -0,0 +1,43 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7'
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: target
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
+    wrappers:
+        - ansicolor
+        - workspace-cleanup
+        - timestamps
+        - build-name:
+            name: ${{ENV,var="name"}}-${{ENV,var="version"}}-${{ENV,var="release"}}
+    publishers:
+        - archive:
+            artifacts: '*.txt, *.xml'
+            allow-empty: 'true'
+        - junit:
+            results: 'J*.xml'
+        - email-ext:
+            recipients: ${{OWNERSHIP,var="JOB_COOWNERS_EMAILS"}}
+            attach-build-log: true
```

Comment:
Sure, thanks!
I'll disable log attachment for MVP v1 since it may need time to gather up and discuss. And we can apply better solution here later.